src/EventGridEmulator/Dockerfile

```diff
@@ -1,9 +1,11 @@
 ﻿FROM mcr.microsoft.com/dotnet/aspnet:7.0-alpine AS base
 WORKDIR /app
-EXPOSE 8080
+EXPOSE 5000
```

Comment:
I think I'd keep it at 8080, I feel it's the usual standard when you can't use the privileged port 80. As of  .net 8 images the default port is not 8080, could be good to follow this aswell. https://learn.microsoft.com/en-us/dotnet/core/compatibility/containers/8.0/aspnet-port